uberfire-structure/uberfire-structure-backend/src/test/java/org/guvnor/structure/backend/repositories/ConfiguredRepositoriesTest.java

```diff
@@ -65,94 +58,101 @@
+                                         "com.space1",
+                                         createAdminContributor("admin1"),
+                                         repositoriesSpace1,
+                                         Collections.emptyList());
+
+        ArrayList<RepositoryInfo> repositoriesSpace2 = new ArrayList<>();
+        repositoriesSpace2.add(addRepository(SPACE2,
+                                             REPO1S2,
+                                             Collections.emptyList(),
+                                             "master"));
+        repositoriesSpace2.add(addRepository(SPACE2,
+                                             REPO2S2,
+                                             Collections.emptyList(),
+                                             "master",
+                                             "dev",
+                                             "release"));
+        repositoriesSpace2.add(addRepository(SPACE2,
+                                             REPO3S2,
+                                             Collections.emptyList(),
+                                             "master",
+                                             "dev",
+                                             "release"));
+
+        SpaceInfo space2 = new SpaceInfo(SPACE2.getName(),
+                                         false,
+                                         "com.space2",
+                                         createAdminContributor("admin2"),
+                                         repositoriesSpace2,
+                                         Collections.emptyList());
+
+        Map<String, SpaceInfo> spaces = new HashMap<>();
+        spaces.put(SPACE1.getName(),
+                   space1);
+        spaces.put(SPACE2.getName(),
+                   space2);
+
+        doAnswer(invocationOnMock -> {
+            final SpaceConfigStorage spaceConfigStorage = mock(SpaceConfigStorage.class);
+            String spaceName = (String) invocationOnMock.getArguments()[0];
+            doReturn(spaces.get(spaceName)).when(spaceConfigStorage).loadSpaceInfo();
+            doReturn(true)
+                    .when(spaceConfigStorage).isInitialized();
+            return spaceConfigStorage;
+        }).when(spaceConfigStorageRegistry).get(any());
+
+        configuredRepositories = new ConfiguredRepositoriesImpl(repositoryFactory,
+                                                                spaceConfigStorageRegistry);
     }
 
-    private ConfigGroup createSpaceConfigGroup(final Space space,
-                                               final List<Contributor> contributors) {
-        final ConfigGroup configGroup = spy(new ConfigGroup());
-        configGroup.setName(space.getName());
-        configGroup.addConfigItem(configurationFactory.newConfigItem("space-contributors", contributors));
-
-        return configGroup;
+    private List<Contributor> createAdminContributor(String... username) {
```

Comment:
I think this should be renamed:
```suggestion
    private List<Contributor> createAdminContributors(String... usernames) {
```